test/unit/services/upload.spec.js

```diff
@@ -79,4 +79,74 @@ describe('uploadService', () => {
         expect(err).toBe('error')
       }
     })
+  it('should update a file in Azure Blob Storage successfully', async () => {
+    const file = {
+      buffer: 'data:image/jpeg;base64,/9j/4AAQSkZJRgABAgAAAQABAAD...',
+      name: 'example.jpg',
+      newName: 'exampleName.jpg'
+    }
+    const blobNameNew = `${file.newName}`
+
+    const fn = (blobUrl, containerName, newBlobName, cb) => {
+      cb(null, newBlobName)
+    }
+    const fnP = (container, blobName, cb) => {
```

Comment:
move these variables above, also as I see, you can reuse it in other tests without creating same variable in different tests

Reply:
fixed